app/controllers/application_controller.rb

```diff
@@ -51,8 +51,8 @@ def check_for_terms_acceptance!
       accepted = current_user.terms_accepted_at
       if accepted.nil?
         raise MissingTOSAcceptance
-      elsif accepted < Document.terms_and_conditions.updated_at
-        raise OutadedTOSAcceptance
+       elsif accepted < Document.terms_and_conditions.updated_at
```

Comment:
adds spaces by mistake?

Reply:
Sorry, it was a mistake, the file has been reverted to develop version.